.github/workflows/deploy-preview.yml

```diff
@@ -0,0 +1,31 @@
+# .github/workflows/deploy-preview.yml
+
+name: Deploy to Preview Channel
+
+on:
+  pull_request:
+
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./example
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "12.x"
+      - uses: subosito/flutter-action@v1
+        with:
+          channel: stable
+      - run: flutter pub get
+      - run: flutter build web --release
+      - uses: FirebaseExtended/action-hosting-deploy@v0
+        with:
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
+          firebaseServiceAccount: "${{ secrets.FIREBASE_SERVICE_ACCOUNT_ANIMATED_TEXT_KIT }}"

```

Comment:
💭 `FIREBASE_SERVICE_ACCOUNT_ANIMATED_TEXT_KIT` is long.
Is qualifying a _project secret_ with the project name (`_ANIMATED_TEXT_KIT`) really necessary? 🤔

Reply:
Actually, I also feel the same but this was the recommended way, but I have updated the secret name to `FIREBASE_SERVICE_ACCOUNT`.